Program/program2.py

```diff
@@ -0,0 +1,33 @@
+Python 3.12.4 (tags/v3.12.4:8e8a4ba, Jun  6 2024, 19:30:16) [MSC v.1940 64 bit (AMD64)] on win32
+Type "help", "copyright", "credits" or "license()" for more information.
+>>> import sqlite3 as sq
+... import csv

```

Comment:
Is there another way to import csv file for reading and writing ??

Reply:
yes we can import csv for reading and writing by this " from csv import reader ","from csv import writer"..